### PR TITLE
Task4 Changes 

### PR DESCRIPTION
**Pull request by scutt5**
Here are the changes for Task 4. Thanks!

### CODE REVIEW

File: starter/src/main/java/com/starter/fullstack/rest/InventoryController.java [hunk 1]
```diff
@@ -3,13 +3,17 @@
 import com.starter.fullstack.api.Inventory;
 import com.starter.fullstack.dao.InventoryDAO;
 import java.util.List;
+import java.util.Optional;
+
```
Comment (JeremiahFailla) [inline starter/src/main/java/com/starter/fullstack/rest/InventoryController.java]: remove empty line

File: starter/src/main/java/com/starter/fullstack/rest/InventoryController.java [hunk 2]
```diff
@@ -41,11 +45,25 @@ public List<Inventory> findInventories() {
...
+   * @param id the inventory id as a String.
+   * @return the deleted inventory.
+   */
+  @DeleteMapping
+  public Inventory deleteInventory(@RequestBody String id){
+    Optional<Inventory> ret = this.inventoryDAO.delete(id);
+    if(ret.isPresent()){
+      return ret.get();
+    }
+    //else is empty
```
Comment (JeremiahFailla) [inline starter/src/main/java/com/starter/fullstack/rest/InventoryController.java]: remove comment

**Review by JeremiahFailla — CHANGES_REQUESTED** [review]
Looks mostly good just some minor feedback. Also missing the unit tests.

**Comment by scutt5** [comment]
Should have the missing unit tests now and the previous feedback.

File: starter/src/main/java/com/starter/fullstack/rest/InventoryController.java (hunk 1 above)
Comment (JeremiahFailla) [inline starter/src/main/java/com/starter/fullstack/rest/InventoryController.java]: looks like you still have this empty line. Please remove it. There shouldn't be any empty lines in your imports. Can you also remove the empty line you have in the imports in InventoryDAO.java

File: starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java [hunk 3]
```diff
@@ -69,5 +69,20 @@ public void create() throws Throwable {
...
+  /**
+   * Test delete endpoint.
+   * @throws Throwable see MockMvc
+   */
+  @Test
+  public void remove() throws Throwable {
+    this.mockMvc.perform(delete("/inventory")
+        .accept(MediaType.APPLICATION_JSON)
+        .contentType(MediaType.APPLICATION_JSON)
+        .content("[\"" + this.inventory.getId() + "\"]"))
```
Comment (JeremiahFailla) [inline starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java]: This should be a single id not a list

File: starter/src/main/java/com/starter/fullstack/rest/InventoryController.java (hunk 2 above)
Comment (JeremiahFailla) [inline starter/src/main/java/com/starter/fullstack/rest/InventoryController.java]: Dont forget to remove this comment

**Review by JeremiahFailla — CHANGES_REQUESTED** [review]
Minor feedback

**Review by JeremiahFailla — APPROVED** [review]

**Pull request merged**